test/unittest/unittest_nntrainer_layers.cpp

```diff
@@ -1789,13 +1808,13 @@ TEST(nntrainer_ActivationLayer, forward_backward_01_p) {
                  nntrainer::ActivationLayer::relu((l - 4) * 0.1 * (i + 1)));
   nntrainer::Tensor result;
   EXPECT_NO_THROW(result =
-                    layer.forwarding(MAKE_SHARED_TENSOR(input)).get()[0]);
+                    layer.forwarding({MAKE_SHARED_TENSOR(input)})[0].get()[0]);
```

Comment:
Because shared_ptr shouldn't be interpreted as an array from this PR, I guess it is worth changing `sharedTensor.get()[0]` to `*sharedTensor` as well :)

and for others parts! 
```suggestion
                    *layer.forwarding({MAKE_SHARED_TENSOR(input)})[0]);
```

Reply:
We are using both in other codes. We have to find and change it separate pr.